src/CompatibilityMessageConverter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This file is part of the prooph/pdo-event-store.
+ * (c) 2016-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Pdo;
+
+use Prooph\Common\Messaging\Message;
+use Prooph\Common\Messaging\MessageConverter;
+
+/**
+ * @deprecated This class is used for backwards compatibility. It's recommended to use a different MessageConverter implementation such as \Prooph\Common\Messaging\NoOpMessageConverter.
+ */
+class CompatibilityMessageConverter implements MessageConverter
```

Comment:
Why not use `NoOpMessageConverter` directly? What is this class for? Why is it deprecated?

Reply:
It's deprecated because such class does not belong to this package and is only here for BC.

NoOpMessageConverter should be located in prooph/common. It is already there but requires DomainMessage which is a slightly different case.

I can add the converter to prooph/common, use it here and remove this class - of course it will require increase in required prooph/common version which technically means losing BC. Are you fine with that? If so, any tip for the class name since NoOpMessageConverter is already taken?